qcsrc/spectate.qc

```diff
@@ -0,0 +1,111 @@
+/*
+	spectate.qc
+
+	spectator functions
+
+	PURPOSE
```

Comment:
"PURPOSE"?

Reply:
This is just verbatim from the original progs. What do you think should be changed?

Reply:
Seems it was leftover since the purpose line is above it. Can you remove this like to be tidy?